js/Control.OSMGeocoder.js

```diff
@@ -19,7 +19,7 @@ L.Control.OSMGeocoder = L.Control.extend({
 			this._map.fitBounds(bounds);
 
 			// IDE - Map have the power to draw a marker
-			RoadMap.addMarkerSearch(this._map.fitBounds(bounds));
+			RoadMap.setRequestedPositionMarker(this._map.fitBounds(bounds));
```

Comment:
If you simply want to rename the function, you can also define an alias in the code :)

As I think this vendor code was already modified in the past, I think it is reasonable to open an issue and revert all the vendor code later.